dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Qdrant/DotNet/IVectorDb.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Threading.Tasks;
+
+namespace Qdrant.DotNet;
```

Comment:
1) DotNet feels odd -- do we ever expect a different folder under QDrant that isn't DotNet code?
2) My gut says this should be part of Microsoft.SemanticKernel.Skills.Memory.VectorDB.Qdrant namespace

Reply:
I agree. It's already under the dotnet src. In other languages, you can imagine this being under the python/src or typescript/src.